pkg/rancher-desktop/utils/_demo_marketplace_items.js

```diff
@@ -172,14 +172,14 @@ export const demoMarketplace = {
     },
     {
       id:                    '',
-      name:                  'Blockly Automation',
+      name:                  'Blockly Automation', // spellcheck-ignore-line
```

Comment:
We should probably ignore the whole file, but this is fine for now.

Reply:
We can ignore the whole file with a single entry in `.github/actions/spelling/excludes.txt`